tests/drivers/clock_control/stm32_clock_configuration/stm32_common_devices/src/test_stm32_clock_configuration.c

```diff
@@ -39,12 +39,51 @@ ZTEST(stm32_common_devices_clocks, test_sysclk_freq)
 #define STM32_I2C_DOMAIN_CLOCK_SUPPORT 0
 #endif
 
+static void i2c_set_clock(struct stm32_pclken clock)
```

Comment:
You should rather pass a `const struct stm32_pclken *clock` here

Reply:
@arbrauns Can you take  this comment into account?
This is the only blocking comment imo

Reply:
Done.